modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi

```diff
@@ -0,0 +1,60 @@
+@App:name('ReadFileInBinaryChunkedMode')
+
+@App:description('Read the file as binary chunks')
```

Comment:
Make this as "Read a binary file as chunks"

Reply:
fixed in https://github.com/wso2/streaming-integrator-tooling/pull/87/commits/3b0b69ece28668ca58eaee3041d4cb7636494296